.github/workflows/docs.yml

```
@@ -0,0 +1,19 @@
+name: Docs
+
+on:
+  pull_request_target:
+    branches:
+      - main
+
+jobs:
+  verify:
+    runs-on: [self-hosted]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Run check format
+        run: npm run check:format
```

```suggestion
        run: cd docs && npm run check:format
```

You need to cd into `docs` before running any command.